great_ape_safe/ape_api/ebtc.py

```diff
@@ -140,11 +167,20 @@ def cancel_lowsec_timelock(
         predecessor=None,
         salt=None,
```

Comment:
Great job w/ the flexibility of passing either `id` or params into the timelock methods for cancelling

However, will imagine that tracking all those params + unique salt will be tricker at the time of exec the script, so imagine that using only the **id** may be sufficient and the preferred approach, but having the flex is great

how are you envisioning the tracking of the salts used to avoid repeated queue actions crashing on the same id? (i acknowledge that it may have low %, but curious on the approach you have in mind)

Reply:
My idea for tracking all activity on the Timelocks is to build a UI service for displaying all timelock transactions with all their details including id and salts. I am scoping this out but I agree with your concern that it will be a pain to review and manage timelock operations without access to this info.

Reply:
the ui is def a great addition, something that came to my mind that will be valuable to support is that when queuing a tx in the timelock to have a better scope of what storage variables may affect or what modifications represents, would be worth to have a tenderly simulation url to share w/ the rest of potential signers? 

wdyt of adding that feature in the flow while either queuing and/or executing so it prints out on the cli?

Reply:
resolving this comment in light of this ticket https://github.com/ebtc-protocol/ebtc-multisig/issues/16 to be work separaetly so it does not get too dense this pr